src/Posts.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { useEffect, useState } from 'react';
```

Comment:
what is it?

Reply:
I will move it to another project. Thanks.